dev-board/Src/imu-driver/bno055.h

```diff
@@ -0,0 +1,162 @@
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdarg.h>
+
+#ifndef BNO055_H_
+#define BNO055_H_
+
+#define BNO055_ADDRESS 0x28
```

Comment:
Can you split up your code into different sections like https://github.com/Solar-Gators/Telemetry_2019-2020/blob/master/templates/c-header-template.h#L8 for stuff like include files, defines, structs, public declartions etc